portfolio/src/main/webapp/index.html

```diff
@@ -174,8 +174,44 @@ <h4>
         <div class="info">     
           <h2>Projects</h2>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
+          <br>
```

Comment:
Avoid using br for presentational aspects such as margin - use CSS for that. HTML is only for structure and semantics.